diesel_derives/src/associations.rs

```diff
@@ -147,13 +147,29 @@ impl AssociationOptions {
             .skip(1)
             .filter(|n| !n.name().is_ident("foreign_key"));
         for ignored in unrecognized_options {
-            ignored
-                .span()
-                .warning(format!(
-                    "Unrecognized option {}",
-                    path_to_string(&ignored.name())
-                ))
-                .emit();
+            match ignored.path() {
+                Ok(_) => {
+                    ignored
+                        .span()
+                        .warning(
+                            "belongs_to takes a single parent. Change\n\
+                            \tbelongs_to(Parent1, Parent2)\n\
```

Comment:
Any possibility that this error message contains the actual parameters passed to the attribute, so that for `#[belongs_to(Bar, Baz)]` the following warning is emitted:

```
warning: belongs_to takes a single parent. Change
    belongs_to(Bar, Baz)
to
    belongs_to(Bar)
    belongs_to(Baz)
````